### DISCUSSION TITLE
Scroll bar appearing on wallpapers

### DISCUSSION THREAD

**Discussion by Micromic0** [rocksdanister/lively#1963] [General]
**Describe the bug**
If I choose any theme and stretch it out to both of my displays, two scroll bars appear: on the first display it appears on the bottom of it, and on the second one it appears on the side of the side of it.

**To Reproduce**
1. Open the app
2. Choose any theme
3. Stretch it to both displays 
4. See both scroll bars

**Expected behavior**
No scrollbar appears.

**Screenshots/Video**
1st display (main):
![image](https://github.com/rocksdanister/lively/assets/117858567/7bb09916-a1b1-468f-8f57-9d2691123859)
2nd display:
![image](https://github.com/rocksdanister/lively/assets/117858567/d4704b49-7c24-4c5e-9b9d-24873a09c6e4)

**Desktop (please complete the following information):**
 - OS: Windows 11
- Triangles & Light

**Additional context**
This happened after I restarted my laptop.

**Log file (Important)**
[lively_log_20231026_131016.zip](https://github.com/rocksdanister/lively/files/13175976/lively_log_20231026_131016.zip)

**Comment by rocksdanister** [comment]
Add this to the css style:
https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp

**Reply by Micromic0** [reply]
Thank you!
